src/app/backend/dashboard.go

```diff
@@ -76,14 +77,17 @@ func main() {
 
 	log.Printf("Successful initial request to the apiserver, version: %s", versionInfo.String())
 
-	heapsterRESTClient, err := heapster.CreateHeapsterRESTClient(*argHeapsterHost,
-		apiserverClient)
+	// Init integrations
+	integrationManager := integration.NewIntegrationManager(clientManager)
+	err = integrationManager.Metric().
+		ConfigureHeapster(*argHeapsterHost).
+		Enable(integrationapi.HeapsterIntegrationID)
```

Comment:
hmm...I wonder if this is the right approach. I have not checked the api or anything lately, but my understanding that we do not have to care about which monitoring system is installed. We just use the api facade from apiserver. Also, installing a monitoring system might not be as easy as installing e.g. tiller. 

The old issue: https://github.com/kubernetes/dashboard/issues/1310

However, we should care about the integration of Graphana.